### PR TITLE
PP-2032: Add database metrics to pay-adminusers

### PR DESCRIPTION
**Pull request by whpearson**
Registers some metrics and then pulls in the new data for the metrics whenever
the healthcheck is called.

As the pay-connector work does.

### CODE REVIEW

File: src/main/java/uk/gov/pay/adminusers/app/healthchecks/DatabaseHealthCheck.java [hunk 1]
```diff
@@ -36,4 +83,17 @@ protected Result check() throws Exception {
...
+        statement.execute("select * from pg_stat_database where datname='adminusers';");
+        ResultSet resultSet = statement.getResultSet();
+        resultSet.next();
+        for (String key: integerDatabaseStatsMap.keySet() ) {
+            integerDatabaseStatsMap.put(key, resultSet.getInt(key));
+        }
+        for (String key: doubleDatabaseStatsMap.keySet() ) {
+            doubleDatabaseStatsMap.put(key, resultSet.getDouble(key));
+        }
+    }
```
Comment (kakumara) [inline src/main/java/uk/gov/pay/adminusers/app/healthchecks/DatabaseHealthCheck.java]: Its normally a good practice to close statements and resultsets after use. 
```
resultSet.close();
statement.close();
```

**Comment by kakumara** [comment]
Other than that 👍

File: src/main/java/uk/gov/pay/adminusers/app/healthchecks/DatabaseHealthCheck.java (hunk 1 above)
Comment (whpearson) [inline src/main/java/uk/gov/pay/adminusers/app/healthchecks/DatabaseHealthCheck.java]: Will do.

**Comment by whpearson** [comment]
I've done a different change that has the same effect. The try ( foo=bar) { } syntax closes things that are auto-closeable at the end of it.

**Comment by maxcbc** [comment]
👍 cool then merge away :)

**Pull request merged**